tools/influx_si/README.rst

```diff
@@ -4,17 +4,17 @@ Metabolic flux estimation
 
 `influx_si` can be used for metabolic flux estimation based on labeling data from MS and/or NMR measurements. `influx_s` is used for stationary labeling and `influx_i` for instationary one (hence the '_s' and '_i' in the names). Tha name `influx_si` is used as common for both tools.
 
-The input for `influx_si` in one or several FTBL files, each describing a given labeling experiment and options to use during estimation. In addition, for `influx_i` a file with time course data has to be provided for each FTBL file.
+The input for `influx_si` in one or several MTF (Multiple Text Files) collections, each describing a given labeling experiment and options to use during estimation.
 
-The output is a zip archive with files generated by `influx_si`. Among the most important is file ending with `_res.kvh` which contains the main information about estimated parameters, their statistics and so on.
-If, in your FTBL file, you have requested ploting information, it can be found in respective .pdf file.
+The output is a zip archive with files generated by `influx_si`. Among the most important are files ending with `.sim` (as 'simulated') which contains information about estimated parameters, their SD and so on.
+If, in your .opt file, you have requested plotting information, it can be found in respective .pdf file.
 
-For detailed documentation about `influx_si` and accompanying tools see https://metasys.insa-toulouse.fr/software/influx/doc/
+For detailed documentation about `influx_si` and accompanying tools see https://influx-si.readthedocs.io
 
-For getting a standalone version of `influx_si` see https://metasys.insa-toulouse.fr/software/influx/
+For getting a standalone version of `influx_si` see https://anaconda.org/bioconda/influx_si
 
 Author: Serguei Sokol
```

Comment:
you can use https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-creator-person instead

Reply:
README.rst and influx_si.xml (with <creator> tag) are not visible in the same places. So I kept "Author" in README.rst and added <creator> to .xml